.circleci/config.yml

```diff
@@ -152,6 +154,11 @@ jobs:
           no_output_timeout: 30m
           command: |
             cd ./monitor-ci; docker-compose -f compose/fig.cloud.yml -f compose/fig.cypress.yml run cypress ./node_modules/.bin/cypress run --browser chrome --config numTestsKeptInMemory=1 --spec "$(cd ../ui; circleci tests glob "./cypress/e2e/**/*.test.ts" | circleci tests split | paste -sd "," -)"
+      - run:
+          name: Push IDPE UI E2E images
+          command: |
```

Comment:
// need to add something like this
      - when:
          condition:
            equal: [ "master", "<< pipeline.git.branch >>" ]